src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationEventListenerIntTest.kt

```diff
@@ -478,17 +453,6 @@ class ProbationEventListenerIntTest : MessagingMultiNodeTestBase() {
     }
   }
 
-  @ParameterizedTest
-  @MethodSource("domainEvents")
-  fun `should process domain events`(event: String) {
-    val crn = randomCrn()
-    stubPersonMatchUpsert()
-    stubPersonMatchScores()
-    probationDomainEventAndResponseSetup(event, ApiResponseSetup(crn = crn))
-    checkTelemetry(CPR_RECORD_CREATED, mapOf("SOURCE_SYSTEM" to "DELIUS", "CRN" to crn))
-    checkEventLogExist(crn, CPRLogEvents.CPR_RECORD_CREATED)
-  }
```

Comment:
these events are all treated the same - no code looks at which of them we are processing